test/functional/fixtures/api/es-next/take-screenshot/testcafe-fixtures/take-element-screenshot.js

```diff
@@ -98,6 +100,33 @@ test
             .takeElementScreenshot('table', 'custom/' + t.ctx.parsedUA.family + '.png');
     });
 
+test
+    .page('../pages/nested-iframe.html')
+    ('Rescroll parents', async t => {
+        await t
+            .switchToIframe('iframe')
+            .switchToIframe('iframe')
+            .takeElementScreenshot('table', 'custom/' + t.ctx.parsedUA.family + '.png');
+
+        await t.switchToMainWindow();
+
+        var internalScrollEventsCount = await getInternalScrollEventsCount();
+
+        await t.expect(internalScrollEventsCount).lte(1);
```

Comment:
we can use `ClientFunctions` in assertions directly, e.g.
`await t.expect(getInternalScrollEventsCount()).lte(1);`